testing/architecture-test/build.gradle.kts

```diff
@@ -64,6 +65,12 @@ tasks {
 
         systemProperty("org.gradle.public.api.includes", (PublicApi.includes + PublicKotlinDslApi.includes).joinToString(":"))
         systemProperty("org.gradle.public.api.excludes", (PublicApi.excludes + PublicKotlinDslApi.excludes).joinToString(":"))
+
+        val platformsDataTask = rootProject.tasks.named("platformsData")
+        dependsOn(platformsDataTask)
+        systemProperty("org.gradle.architecture.platforms-json", platformsDataTask.get().outputs.files.singleFile.absolutePath)
+        systemProperty("org.gradle.architecture.platforms-base-path", layout.settingsDirectory.file("platforms").asFile.absolutePath)
```

Comment:
❓ Shouldn't the referenced files be added to task inputs? I think this task checks for up-to-dateness.

Reply:
Fixed using a command line arg provider